integration-tests/src/main/java/org/apache/druid/testing/utils/KinesisAdminClient.java

```diff
@@ -156,15 +162,16 @@ public boolean verfiyPartitionCountUpdated(String streamName, int oldShardCount,
     return actualShardCount == oldShardCount + newShardCount;
   }
 
-
-  private boolean verifyStreamStatus(StreamDescription streamDescription, StreamStatus streamStatusToCheck)
```

Comment:
You could retain a modified `verifyStreamStatus` method instead of the doing the String.equals() everywhere.